fdb-relational-grpc/fdb-relational-grpc.gradle

```diff
@@ -65,13 +65,11 @@ dependencies {
     implementation(libs.log4j.api)
     runtimeOnly(libs.log4j.core)
     // slf4j binding. Allows RL dependencies to log to same place as Relational
-    implementation("org.apache.logging.log4j:log4j-api:${log4jVersion}")
-    runtimeOnly("org.apache.logging.log4j:log4j-core:${log4jVersion}")
-    runtimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}")
-    implementation("com.google.protobuf:protobuf-java:${protobufVersion}")
-    testImplementation("org.junit.jupiter:junit-jupiter-api:${junitVersion}")
-    testImplementation("org.junit.jupiter:junit-jupiter-params:${junitVersion}")
-    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:${junitVersion}")
+    runtimeOnly(libs.log4j.slf4jBinding)
+
+    testImplementation(libs.bundles.test.impl)
+    testRuntimeOnly(libs.bundles.test.runtime)
+    testImplementation(libs.grpc.testing)
```

Comment:
Huh, I'm surprised you don't need junit in here, but it ran the tests, so I guess it's good.